docs/providers/eveonline.md

```diff
@@ -53,7 +53,7 @@ options: {
       }
       return token;
     },
-    session: async (session, token) => {
+    session: async ({ session, token }) => {
       if (token) {
         session.user.id = token.id;
       }
```

Comment:
You can also drop the if statement here.
```suggestion
    session: async ({ session, token }) => {
      session.user.id = token.id;
```

Reply:
Noice. Didn't want to completely rewrite away from the initial style. Once I get the EVE stuff up and running I'd be happy to update this further.